lib/SecureStorage.ts

```diff
@@ -0,0 +1,72 @@
+import * as ExpoSecureStore from "expo-secure-store"
+
+export type SecureStorage = typeof ExpoSecureStore
```

Comment:
```suggestion
export interface SecureStore {
  getItemAsync (key: string): Promise<string>
  deleteItemAsync (key: string): Promise<void>
  setItemAsync (key: string): Promise<void>
}
```

This is so that you don't need to mock all the irrelevant methods (like `isAvailiableAsync`) for testing.